### PR TITLE
[FrameworkBundle] Add missing autowiring aliases for common interfaces

### PR DESCRIPTION
**Pull request by chalasr**
| Q             | A
| ------------- | ---
| Branch?       | master
| Bug fix?      | no
| New feature?  | yes
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | -
| License       | MIT
| Doc PR        | n/a

### CODE REVIEW

**Comment by nicolas-grekas** [comment]
should we add LoggerInterface also? or do we consider it should be done on monolog bundle?

**Comment by chalasr** [comment]
> do we consider it should be done on monolog bundle?

Not sure but I would say yes so each bundle is responsible for supporting autowiring for its services

**Review by nicolas-grekas — APPROVED** [review]
:+1:

**Comment by fabpot** [comment]
Thank you @chalasr.

File: src/Symfony/Bundle/FrameworkBundle/Tests/Functional/app/AutowiringTypes/bundles.php [hunk 1]
```diff
@@ -11,8 +11,10 @@
 
 use Symfony\Bundle\FrameworkBundle\Tests\Functional\Bundle\TestBundle\TestBundle;
 use Symfony\Bundle\FrameworkBundle\FrameworkBundle;
+use Symfony\Bundle\SecurityBundle\SecurityBundle;
 
 return array(
     new FrameworkBundle(),
+    new SecurityBundle(),
```
Comment (xabbuh) [inline src/Symfony/Bundle/FrameworkBundle/Tests/Functional/app/AutowiringTypes/bundles.php]: Why is this test located in the FrameworkBundle and not in the SecurityBundle?

Comment (chalasr) [inline src/Symfony/Bundle/FrameworkBundle/Tests/Functional/app/AutowiringTypes/bundles.php]: @xabbuh I thought about that but it was simpler to integrate a new test here than duplicating the whole test case on the SecurityBundle. Does it hurt?

Comment (fabpot) [inline src/Symfony/Bundle/FrameworkBundle/Tests/Functional/app/AutowiringTypes/bundles.php]: I missed that, but indeed, that's not good. @chalasr Can you move it to SecurityBundle?

Comment (chalasr) [inline src/Symfony/Bundle/FrameworkBundle/Tests/Functional/app/AutowiringTypes/bundles.php]: Ok I'll do

Comment (chalasr) [inline src/Symfony/Bundle/FrameworkBundle/Tests/Functional/app/AutowiringTypes/bundles.php]: See https://github.com/symfony/symfony/pull/21522

**Comment by chalasr** [comment]
@nicolas-grekas See https://github.com/symfony/monolog-bundle/pull/203 for LoggerInterface

**Comment by stof** [comment]
I would also ensure that ``translator`` can be autowired properly (we also decorate it in dev)

**Pull request merged**